content/altinn-studio/reference/monitoring/configuration/_index.en.md

```diff
@@ -40,14 +40,42 @@ This is relevant if you
 * Customize sampling
 * Need to export telemetry to a different backend
 
-In this case, the configuration only adds an additional exporter to the respective provider-builders.
+In this case, the configuration only adds an additional exporter (`ConsoleExporter`) to the respective provider-builders.
+
+First, we update the `App.csproj` file:
+
+{{< highlight csproj "linenos=false" >}}
+        <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.9.0" />
+{{< / highlight >}}
+
+Then we update the configuration code in `Program.cs`:
+
+{{< highlight csharp "linenos=false,hl_lines=1-3 7-25" >}}
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
 
-{{< highlight csharp "linenos=false,hl_lines=3-5" >}}
 void RegisterCustomAppServices(IServiceCollection services, IConfiguration config, IWebHostEnvironment env)
 {
-    services.ConfigureOpenTelemetryMeterProvider(builder => builder.AddConsoleExporter());
-    services.ConfigureOpenTelemetryTracerProvider(builder => builder.AddConsoleExporter());
-    services.Configure<OpenTelemetryLoggerOptions>(builder => builder.AddConsoleExporter());
+    // Configure Console export across OTel providers, just for demonstration purposes.
```

Comment:
nit: (clarify+rm uppercase c) // Configure exporters for telemetry signals (metrics, traces and logging). Here console exporters are used for demonstration purposes